src/components/NavbarStudy.scss

```diff
@@ -0,0 +1,158 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+$primary-color: rgb(82, 105, 207);
+$white-color: #fff;
+$content-color: #1B1B1F;
+
+.menu {
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    top: 50px;
+    left: 0;
+}
+
+.blur {
+    position: absolute;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    backdrop-filter: none;
+    transition: .2s;
+}
+
+.blur.active {
+    backdrop-filter: blur(2px);
+}
+
+.menu__content {
+    position: relative;
+    width: 64px;
+    height: 794px;
+    background-color: $white-color;
+    border-top: 1px solid rgba(0, 0, 0, 0.2);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+    border-right: 1px solid rgba(0, 0, 0, 0.2);
+    box-shadow: 5px 10px 10px rgba(0, 0, 0, 0.5);
+    transition: .1s;
+}
+
+.menu__content.active {
+    width: 200px;
+    border-radius: 0 15px 15px 0;
+    transition: .1s;
+}
+
+.menu__inner {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    padding-top: 26px;
+}
+
+.menu__item {
+    display: flex;
+    justify-content: start;
+    align-items: center;
+    list-style-type: none;
+    width: 100%;
+    margin: 0 0 30px 15px;
+}
+
+.menu__item-inner {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+}
+
+.menu__item-inner:hover .menu__text {
+    color: $primary-color;
+}
+
+.menu__item-inner:hover .svg {
+    fill: $primary-color;
+}
+
+
+.menu__text {
+    width: 0;
+    text-align: left;
+    font-size: 0;
+    font-weight: 600;
+    color: $content-color;
+    opacity: 0;
+    transition: .1s;
+}
+
+.menu__text.active {
+    width: 100px;
+    font-size: 1em;
+    opacity: 1;
+    transition: .1s;
+}
+
+.menu__text.selected {
+    color: $primary-color;
+}
```

Comment:
if you're using BEM - use the next naming rule:

`.menu__text.selected` -> `.menu__text--selected`

Reply:
done